pkg/cli/run_test.go

```diff
@@ -52,14 +52,14 @@ func TestRun(t *testing.T) {
 						TypeMeta:   metav1.TypeMeta{},
 						ObjectMeta: metav1.ObjectMeta{Name: "found"},
 						Spec:       v1.AppInstanceSpec{Secrets: []v1.SecretBinding{{Secret: "found.secret", Target: "found"}}},
-						Status:     v1.AppInstanceStatus{Ready: true},
+						Status:     apiv1.AppStatus{Ready: true},
 					}, nil
 				case "found.container":
 					return &apiv1.App{
 						TypeMeta:   metav1.TypeMeta{},
 						ObjectMeta: metav1.ObjectMeta{Name: "found.container"},
 						Spec:       v1.AppInstanceSpec{Secrets: []v1.SecretBinding{{Secret: "found.secret", Target: "found"}}},
-						Status:     v1.AppInstanceStatus{},
+						Status:     apiv1.AppStatus{},
```

Comment:
nit: these zero-valued `Status` fields can all be removed.

Reply:
done